clipster

```diff
@@ -458,7 +462,7 @@ class Daemon(object):
             return
 
         # Only monitor owner-change events for apps with certain WM_CLASS values
-        if self.whitelist_classes:
+        if not isinstance(self.whitelist_classes, str):
```

Comment:
I don't really like this, because the type of whitelist_classes can either be string if empty, or list if non empty. But it works. What do you think?